src/parser/SGTypes.ts

```diff
@@ -291,8 +291,8 @@ export class SGElement {
 
     public transpile(state: TranspileState) {
         return new SourceNode(null, null, null, [
-            state.transpileToken(this.tokens.startTagOpen, '<'), // <
-            state.transpileToken(this.tokens.startTagName),
+            ...state.transpileToken(this.tokens.startTagOpen, '<'), // <
```

Comment:
the `source-map` library accepts infinitely recursive arrays of arrays of arrays.... so we don't need to do the spread on any of these. I'd rather not do the spread, because `source-map` runs in web-assembly so it'll actually be more efficient for it to traverse the nested arrays. 

So let's undo all the spreads for `state.transpileToken()` and just pust the resulting array there